vulture/utils.py

```diff
@@ -56,6 +56,18 @@ def format_path(path):
     return relpath if not relpath.startswith('..') else path
 
 
+def get_decorator_name(decorator):
+    name = ''
+    if isinstance(decorator, ast.Call):
+        decorator = decorator.func
+    while isinstance(decorator, ast.Attribute):
+        name = decorator.attr + ('.' + name if name else '')
+        decorator = decorator.value
+    else:
+        name = decorator.id + ('.' + name if name else '')
```

Comment:
This statement will always be executed, right, because the "while" loop has no "break"? Then we should omit "else:".

Reply:
Oh, yes!